spec/graphql/mutations/invoices/finalize_spec.rb

```diff
@@ -44,4 +45,35 @@
       end
     end
   end
+
+  context 'with tax provider' do
+    let(:integration) { create(:anrok_integration, organization:) }
+    let(:integration_customer) { create(:anrok_customer, integration:, customer:) }
+
+    before do
+      integration_customer
+    end
+
+    it 'returns pending invoice' do
+      freeze_time do
```

Comment:
do you need to `freeze_time`? there are no time checks

Reply:
updated